include/data_structures/message.hpp

```diff
@@ -8,24 +8,19 @@
 using namespace std;
 
 /*******************************************/
-/**************** Message_t ****************/
+/**************** message_t ****************/
 /*******************************************/
-struct Message_t{
 
-  Message_t(){}
-  Message_t(float i_value)
-   :value(i_value){}
-
-  	float   value;
-
-  void clear() {
-   Message_t(); 
-  }
+struct message_t {
+	float value;

```

Comment:
Kindly review this portion for indentation(4 Spaces).

Reply:
Many thanks for letting us know on the indentation. We have fixed it now. May I request you to please review them?